### PR TITLE
Create Kodiak config.yaml

### PR DESCRIPTION
**Pull request by thedoc31**
Adds config for MWPW-143712 and MWPW-143708

### CODE REVIEW

File: .kodiak/config.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          name: raga
+        customfield_11800: MWPW-140779 # Jira epic security tickets will be assigned to.
+        customfield_11900:
+          value: "Unicorn" # Put your team name here; it must be a valid team in Jira.
+        watchers: # Everyone who is an admin on your repository should be a watcher.
+          - casalino
+          - jmichnow
+          - mauchley
+          - hwong
+          - skamat
```
Comment (sukamat) [inline .kodiak/config.yaml]: @thedoc31 This should be `sukamat` :)

**Review by sukamat — CHANGES_REQUESTED** [review]

File: .kodiak/config.yaml (hunk 1 above)
Comment (thedoc31) [reply .kodiak/config.yaml]: @sukamat So remove honwai? Us ops folks need to keep an eye on the tickets so we're fine :)

Comment (sukamat) [reply .kodiak/config.yaml]: No. I meant the id should be `sukamat` and not `skamat`

Comment (thedoc31) [reply .kodiak/config.yaml]: Fixed!

**Pull request merged**